.circleci/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apt-get install -y \
     tar \
     zlib1g-dev
 
-RUN curl -o /usr/bin/solc -fL https://github.com/ethereum/solidity/releases/download/v0.6.6/solc-static-linux \
+RUN curl -o /usr/bin/solc -fL https://github.com/ethereum/solidity/releases/download/v0.6.1/solc-static-linux \
```

Comment:
Can we set the version as a build variable?

Reply:
yep, but that should be done in a separate PR